src/components/ftbot/TradeList.vue

```diff
@@ -170,6 +170,31 @@ const onRowClicked = ({ data: item }) => {
   }
 };
 
+/**
+ * Calculate the remaining (= not filled) amount of a trade.
+ * Aims at partial fills.
+ * @param orders - array of orders for this trade
+ * @returns {number}
+ */
+const calcRemaining = (orders?: Trade['orders']): number => {
+  if (!Array.isArray(orders)) return 0;
+  return orders.reduce((acc, order) => acc + (order.remaining ?? 0), 0);
+};
+
+/**
+ * Format the amount of a trade.
+ * If there are remaining orders, show the filled / requested amount
+ * @param amount - the actual filled amount
+ * @param orders - array of orders for this trade
+ * @returns {string}
+ */
+const formatAmountWithRemaining = (amount: number, orders?: Trade['orders']): string => {
+  const remaining = calcRemaining(orders);
```

Comment:
This should be more nuanced.
if you don't have open orders - there's no point in calculating this and will only waste cpu cycles.

considering the same logic is used for closed trades (as well as backtesting) - you shouldn't underestimate the performance impact of this - especially if you've got tons of orders on individual trades.
it MAY run fine in your browser - but others might have a less powerfull system.